spec/query_spec.rb

```diff
@@ -70,6 +70,60 @@
     end
   end
 
+  describe "#start_time=" do
+    before do
+      subject.pattern    = "semla"
+      subject.start_time = time
+    end
+
```

Comment:
Why not. I don't think they need their own it block though. I'll add them to the "should convert to UTC" block.